schema/helper-types.asn

```diff
@@ -84,6 +84,15 @@ BEGIN
       ...
    }
 
+   Translation ::= SEQUENCE OF SEQUENCE {
```

Comment:
Let's clarity the rationale of this PR first then get back to review this schema.